app/tests/utils/db_utils.py

```diff
@@ -0,0 +1,33 @@
+from functools import reduce
+from typing import Any, Callable, Tuple
+
+from app.tests.client.db_client import MSSQLClient
+
+NULL_STR = "NULL"
+
+
+def create_sql_values_string(values: Tuple[Any, ...]) -> str:
```

Comment:
we have utilities like this in database/database.py
is this for the sake of keeping the test utils different? I would prefer that it wasn't and that we had these utilities available project-wide, but I'm open to discussion

Reply:
Yeah, just to keep it separate. I don't mind putting it in the regular utils. Does this code still need to be tested?

Reply:
We should keep things separate between testing and actual code base. I agree with @rayraykay.